files/teaching-demo-rubric.md

```diff
@@ -3,28 +3,28 @@ This rubric is provided as a guide for Trainers evaluating potential new instruc
 Trainer. As such, deviation from this rubric is encouraged as needed to accurately assess the trainee's preparation and instructional
 skills.
 
-In general, trainees who have three or more marks in the "Negative" columns below should be asked to redo their demonstration. Even one 
-mark in a "Negative" column can be justification for asking a trainee to redo their demonstration if the problem is significant. As always,
+In general, these demonstrations have a very low (<5%) failure rate. When a trainee is asked to redo a demo, it is simply a matter of correction -- an error to embrace and to learn from -- not rejection. Trainees who have three or more marks in the "Negative" columns below should probably be asked to redo their demonstration. Even one 
```

Comment:
These points are really good to add!

Reply:
- would "have a very low failure rate (less than 5%)" be easier for non-native English speakers to parse?
- rather than "simply a matter of correction", prefer "usually a matter of correction"
- rather than "probably be asked to redo their demonstration", "probably be asked to try again" (to avoid repeating "redo")